<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Runner</class>
 <widget class="QDialog" name="Runner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="compilerLayout">
     <item>
      <widget class="QLabel" name="compileLabel">
       <property name="text">
        <string>Compiler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="compileLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sourceLayout">
     <item>
      <widget class="QLabel" name="sourceLabel">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sourceLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="argsLayout">
     <item>
      <widget class="QLabel" name="argsLabel">
       <property name="text">
        <string>Arguments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="argsLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="runLayout">
     <item>
      <widget class="QLabel" name="runLabel">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="runLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="console"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="compileButton">
       <property name="text">
        <string>Compile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
